stories/dropdown/index.js

```diff
@@ -3,9 +3,10 @@ import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { withDocs } from 'storybook-readme';
 
-import DropdownReadme from '../../docs/dropdown.md';
+import DropdownReadme from '../../docs/dropdown/dropdown.md';
 import DropdownExample from './dropdownExample';
-import MobileExample, { MobileExampleDoc } from './mobileExample';
+import MobileReadme from '../../docs/dropdown/mobileDropdown.md';
```

Comment:
no need to name it mobileDropdown since it's already inside of the dropdown folder. Just `mobile.md` seems good